workflow/rules/pindel.smk

```diff
@@ -29,26 +29,75 @@ rule pindel:
+        if "bedfile" in config.get("pindel_call", {})
+        else "" + config.get("pindel_call", {}).get("extra", ""),
     log:
-        "cnv_sv/pindel/{sample}_pindel.log",
+        "cnv_sv/pindel/{sample}_pindel_call.log",
     benchmark:
         repeat(
-            "cnv_sv/pindel/{sample}_pindel.benchmark.tsv",
-            config.get("pindel", {}).get("benchmark_repeats", 1),
+            "cnv_sv/pindel/{sample}_pindel_call.benchmark.tsv",
+            config.get("pindel_call", {}).get("benchmark_repeats", 1),
         )
-    threads: config.get("pindel", {}).get("threads", config["default_resources"]["threads"])
+    threads: config.get("pindel_call", {}).get("threads", config["default_resources"]["threads"])
     resources:
-        threads=config.get("pindel", {}).get("threads", config["default_resources"]["threads"]),
-        time=config.get("pindel", {}).get("time", config["default_resources"]["time"]),
-        mem_mb=config.get("pindel", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
-        mem_per_cpu=config.get("pindel", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
-        partition=config.get("pindel", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("pindel_call", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("pindel_call", {}).get("time", config["default_resources"]["time"]),
+        mem_mb=config.get("pindel_call", {}).get("mem_mb", config["default_resources"]["mem_mb"]),
+        mem_per_cpu=config.get("pindel_call", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("pindel_call", {}).get("partition", config["default_resources"]["partition"]),
     container:
-        config.get("pindel", {}).get("container", config["default_container"])
+        config.get("pindel_call", {}).get("container", config["default_container"])
     conda:
         "../envs/pindel.yaml"
     message:
         "{rule}: Detect breakpoints in {wildcards.sample}"
     wrapper:
-        "v0.85.1/bio/pindel/call"
+        "v1.2.0/bio/pindel/call"
+
+
+rule pindel2vcf:
+    input:
+        pindel=expand(
+            "cnv_sv/pindel/{{sample}}_{ext}",
+            ext=[
+                "BP",
+                "CloseEndMapped",
+                "D",
+                "INT_final",
+                "INV",
+                "LI",
+                "RP",
+                "SI",
+                "TD",
+            ],
+        ),
+        ref=config["reference"]["fasta"],
+    output:
+        vcf=temp("cnv_sv/pindel/{sample}.vcf"),
+    params:
+        refname=config.get("pindel2vcf", {}).get("refname", "'Genome Reference Consortium Human Build 38'"),
+        refdate=config.get("pindel2vcf", {}).get("refdate", "20131217"),
+        extra=config.get("pindel2vcf", {}).get("extra", ""),
+    log:
+        "cnv_sv/pindel/{sample}_pindel2vcf.log",
```

Comment:
Should be `"cnv_sv/pindel/{sample}.vcf.log"` with the new standard. Same for `benchmark`.